x-pack/MYTESTS.txt

```diff
@@ -0,0 +1,2380 @@
+ debg Loading config file from "/Users/rashmikulkarni/my_workspace/kibana/x-pack/test/functional/config.js"
```

Comment:
Did you mean to commit this file? I don't think we want this merged to `master`

Reply:
mistake reverted.